python/npcomp/frontends/pytorch/__init__.py

```diff
@@ -0,0 +1,39 @@
+# -*- Python -*-
+# This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import torch
+import _torch_mlir
+from _torch_mlir import _get_mlir
+from _torch_mlir import _op_report
+from _torch_mlir import _liveness_report
+from _torch_mlir import set_debug
+from _torch_mlir import lower_to_std
+
+import json
+
+_torch_mlir._initialize_aten_bindings()
+_torch_mlir.set_debug(False, "")
+
+def get_mlir(t):
+  if not isinstance(t, list):
+    t = [t]
+  return _get_mlir(t)
+
+def op_report(mlir):
+  return json.loads( _op_report(mlir) )
+
+def liveness_report(mlir):
+  return json.loads( _liveness_report(mlir) )
+
+def get_mlir_supported_devices(devkind=None):
+  # FIXME: how do we define our own device?
```

Comment:
Can you include a TODO that is more related to what we discussed (# TODO: Stop borrowing the 'xla' device name and figure out how to define a new one upstream).